org.flatpak.easydict-gtk.yml

```diff
@@ -0,0 +1,46 @@
+app-id: org.flatpak.easydict-gtk
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: easydict-gtk
+modules:
+  - name: poetry-core
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps .
+    sources:
+      - type: archive
+        url: https://github.com/python-poetry/poetry-core/archive/refs/tags/1.0.3.tar.gz
+        sha256: b5a9f8284c685c638796a322ea868843da5e2094c664436e44061e6bcea367ea
+  - name: orjson
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps orjson-3.6.4-cp38-cp38-manylinux_2_24_x86_64.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/e8/da/7fa7aa918788688ef68ed3789f19a54334b4a30b7946e631ea61a7e8a8bb/orjson-3.6.4-cp38-cp38-manylinux_2_24_x86_64.whl
+        sha256: 159e2240fc36720a5cb51a1cbc9905dcb8758aad50b3e7f14f6178ce2e842004
+  - name: tinydb
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps tinydb-4.5.2-py3-none-any.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/c7/a3/130a1949dfcf79915f2fe807947eeb91663d4a0bf8f7f2c0cc8d02d24620/tinydb-4.5.2-py3-none-any.whl
+        sha256: 3c5e5c72c98db07e707be4e25f9e135a8a14b96938e4745b1b7187fec523ff58
+  - name: easydict-gtk
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps --no-build-isolation easydict-gtk-0.4.0.tar.gz
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/8b/92/2a4f246d3f75094b1b38221a37ae343fefa3a819b2507c858197f2cfa98f/easydict-gtk-0.4.0.tar.gz
+        sha256: a6fb89ef4392d5dc0546c7b796520d2d92fc1dd2560524460777832d2c580e8a
+finish-args:
```

Comment:
finish args are usually toward the top, before any module.

Reply:
OK, I will fix that, thanks.